halyard-version

```diff
@@ -1 +1 @@
-operator-0.4.x
+operator-75233080
```

Comment:
Will Halyard be retagged by the build process?

Reply:
There's a Jenkins job for Halyard that generates this type of tag (`operator-<commit hash>`) and also retags to `operator-dev`. 
For development, this repository uses the manifest files in `deploy/operator/...` which point to `operator-dev` tag. During operator build process, Halyard image tag is replaced by the value in this `halyard-version` file.
I have seen that the downside of having hayard version in format `0.4.x` is that is difficult to figure out the next version of Halyard for an operator patch update, so I thought that having the commit hash would be enough.